### ISSUE TITLE
setupWorker still returns SetupWorkerApi instead of SetupWorker in 1.0.1

### ISSUE THREAD

**Issue by grafanauser** [kettanaito/msw#1537]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Reproduction repository

private code i am not allowed to share

### Reproduction steps

see tooltip of intellisense in vs code when hovering over return value of setupWorker

![grafik](https://user-images.githubusercontent.com/47564534/218695848-5f8689de-abe1-4982-b752-55f638c7d23c.png)

### Current behavior

package.json contains:
```"msw": "^1.0.1",```

bla.spec.ts file:
```
import { rest, RestRequest, setupWorker, PathParams, SetupWorker } from "msw";

let server = setupWorker(rest.get(...)); // server has type of SetupWorkerApi instead of SetupWorker
```

### Expected behavior

return type of setupWorker is SetupWorker after v1.0.0

**Comment by grafanauser** [comment]
that was quick. looks good. :)

**Comment by kettanaito** [comment]
## Released: v1.1.0 🎉

This has been released in v1.1.0!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v1.1.0)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/1.1.0)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
